addons/web/static/src/webclient/settings_form_view/settings_form_view_mobile.scss

```diff
@@ -1,6 +1,6 @@
 @include media-breakpoint-down(md) {
 
-    .o_base_settings {
+    .o_base_settings_view .o_form_editable {
```

Comment:
why only .o_form_editable?

Reply:
I don´t think the base_settings view is ever in readonly mode ?